ident/tools/matrixeditor.php

```diff
@@ -143,6 +143,7 @@ function submitAttrs(){
 </div>
 <!-- This is inner text! -->
 <div id="innertext">
+	<h1 class="page-heading"><?= $LANG['ALLOW_PUBLIC_EDITS']; ?></h1>
```

Comment:
I think this is supposed to be "Character Matrix Editor", here and on line 39. These don't appear in the lang files.

Reply:
Fixed in c1e85b3e6.